.github/workflows/sdk-regression.yml

```
@@ -0,0 +1,38 @@
+on:
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-regression:
+    name: Check if needs to run sdk regression
```

```suggestion
    name: Check if needs to run sdk regression
    if: ${{ github.event.issue.pull_request }}
```

run only on issue type = PR, [ref](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment-on-issues-only-or-pull-requests-only)